src/nvim/os/fs.c

```diff
@@ -145,7 +145,19 @@ static bool is_executable_in_path(const char_u *name, char_u **abspath)
     return false;
   }
 
+#ifdef WIN32
+  const char *pathext = getenv("PATHEXT");
```

Comment:
Yes, as is returns `NULL` if the requested environment variable is the empty string. So checking the return value of `r = getenv("VAR")` goes from this:
```c
if (r == NULL || *r == '\0') ...
```
... to just:
```c
if (r == NULL) ...
```